terraform/README.md

```diff
@@ -0,0 +1,59 @@
+# Terraform
+
+## Get started
```

Comment:
[suggestion/blocking] this "getting started" is only for a local setup, which I don't think we currently need, as we are running our dev environments locally with `rails s`

I think it would be helpful to start with the structure of the tf files, and have the local setup below, since that's more informative than anything else. 

also, how do we run it for staging/production? (i personally think i'd like to make it into a GitHub action ultimately so we can automate it as much as possible.)

Reply:
I guess it depends on whether we have to run Terraform ourselves or whether we can automate that via GH Actions.  I do not know how to run staging / production other than running it locally, but I can look into automating that, too, in which case you're right, we wouldn't really need the getting started stuff anymore.

Reply:
I should also mention that a lot of the getting started instructions were notes I was taking for myself as I went, to make sure I knew what I had done on my fantastic voyage.